fluentui_core/src/main/java/com/microsoft/fluentui/compose/ModalPopup.kt

```diff
@@ -122,18 +151,32 @@ fun convertWindowInsetsCompatTypeToWindowInsets(windowInsetsCompatType: Int): Wi
 
 /** Custom compose view for [BottomDrawer] */
 private class ModalWindow(
-    private var onDismissRequest: () -> Unit,
+    private var onDismissRequest: (() -> Unit)? = null,
+    private var properties: PopupProperties,
     private val composeView: View,
+    density: Density,
     saveId: UUID,
+    private val popupLayoutHelper: PopupLayoutHelperImpl = if (Build.VERSION.SDK_INT >= 29) {
```

Comment:
can you add comment here or above the implementation below, why we need separate implementation for different versions?

Reply:
Added in the class definitions below